.github/workflows/publish-package.yml

```diff
@@ -98,7 +98,7 @@ jobs:
       url: ${{ steps.deployment.outputs.page_url }}
     needs: build_and_test_deploy
     runs-on: ubuntu-latest
-    if: inputs.publish
+    if: false
```

Comment:
disabling docs publishing when running the publish action so we can get an alpha of v3 published